hub/config/src/main/java/uk/gov/ida/hub/config/resources/TransactionsResource.java

```diff
@@ -80,7 +80,8 @@ private TransactionDisplayData buildTransactionDisplayData(TransactionConfigEnti
         return new TransactionDisplayData(
                 configData.getSimpleId().orElse(null),
                 configData.getServiceHomepage(),
-                configData.getLevelsOfAssurance());
+                configData.getLevelsOfAssurance(),
+                configData.getHeadlessStartpage());
```

Comment:
what happens if we don't have the headless start page in config?

Reply:
It will be `null`.